cmd/exo/cmd/request.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/exoscale/egoscale"
+)
+
+// asyncRequest if no response expected send nil
+func asyncRequest(cmd egoscale.AsyncCommand, msg string, response interface{}) error {
+	fmt.Print(msg)
+	var errorReq error
```

Comment:
```golang
response := cs.Response(cmd)
cs.AsyncRequestWithContext(gContext, cmd, ...)
return response, errorReq
```

Reply:
I can't return an empty interface because I can't create it in the function.
So It's better to give in parameter and more simple

Reply:
This should work.

```diff
-func asyncRequest(cmd egoscale.AsyncCommand, msg string, response interface{}) error {
+func asyncRequest(cmd egoscale.AsyncCommand, msg string) (interface{}, error) {
+    response := cs.Response(cmd)
     ...
-    return errReq
+    return response, errReq
```